java.lang.NullPointerException when building HTML5

**Describe the bug (REQUIRED)**
"Build HTML5" in editor and "bob build" fails with java.lang.NullPointerException. Editor dont provide any information beside the exception. Bob give this:
```
game % java -jar ~/Downloads/bob.jar build
Working...Reading classes... 1% ...done!
Cleaning engine... ...done!
Reading tasks...java.lang.NullPointerException: Cannot read the array length because "bytes" is null
Cause: java.lang.RuntimeException: java.lang.NullPointerException: Cannot read the array length because "bytes" is null
```
Simple build with "command + B" works fine.


**Expected behavior (REQUIRED)**
Nice error description with infomation about file with error

**Defold version (REQUIRED):**
  - Version  1.4.1

**Platforms (REQUIRED):**
 - Platforms: HTML5
 - OS: MacOs

**Minimal repro case project (OPTIONAL):**
[Archive.zip](https://github.com/defold/defold/files/10449789/Archive.zip)

Bob in verbose mode shows:

```
Caused by: java.lang.NullPointerException: Cannot read the array length because "bytes" is null
	at java.base/java.lang.String.<init>(String.java:1435)
	at com.dynamo.bob.pipeline.ProtoUtil.merge(ProtoUtil.java:30)
	at com.dynamo.bob.pipeline.CollectionBuilder.collectSubCollections(CollectionBuilder.java:65)
	at com.dynamo.bob.pipeline.CollectionBuilder.create(CollectionBuilder.java:119)
	at com.dynamo.bob.Project.createTask(Project.java:429)
	... 6 more
```

--- a/com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/ProtoUtil.java
+++ b/com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/ProtoUtil.java
@@ -26,6 +26,15 @@ import com.google.protobuf.TextFormat;
 public class ProtoUtil {
 
     public static void merge(IResource input, Builder builder) throws IOException, CompileExceptionError {
+        byte[] content = input.getContent();
+        if (content == null) {
+            if (!input.exists()) {
+                throw new CompileExceptionError(input, 0, "Resource does not exist");
+            }
+            else {
+                throw new CompileExceptionError(input, 0, "Resource is empty");
+            }
+        }
         try {
             TextFormat.merge(new String(input.getContent()), builder);
         } catch (TextFormat.ParseException e) {

